### ISSUE TITLE
Simplify transfer_hotspot transaction

### ISSUE THREAD

**Issue by cokes518** [helium/blockchain-core#978]
Current implementation of `transfer_hotspot` has deep dependencies on wallet-api and makes it hard for us to compartmentalize these new features for future work. 

We'd like to remove wallet-api from the equation and further simplify the user experience by removing liveness check and the amount as required fields for the transaction to be processed.

Proposed:
replace `transfer_hotspot_v1` with `transfer_hotspot_v2` that takes the following arguments:
* `owner_signature`
* `owner_address`
* `gateway_address`
* `new_owner_address`

Once submitted to the chain, the owner changes. Hotspot location, gain, elevation do not change.

**Comment by madninja** [comment]
I guess one thing to take into consideration is the effect of scammers selling dead hotspots. Perhaps that's fine but it's putting the responsibility solidly on the buyer here to somehow confirm that this gateway is not a brick

**Comment by cokes518** [comment]
right, but the chain shouldn't be the one to govern/be the decision maker.

**Comment by madninja** [comment]
> right, but the chain shouldn't be the one to govern/be the decision maker.

I agree.. very much.. just wanted it to be very clear

**Comment by abhay** [comment]
Message received and I think we can make sure the UX makes that clear as well.
